tests/js-modules/modules.py

```diff
@@ -1436,15 +1447,24 @@ def run(args):
         args.nodes, args.binary_dir, args.debug_nodes, args.perf_nodes, pdb=args.pdb
     ) as network:
         network.start_and_open(args)
+
+        # Needs to happen early, so later tests can deploy it if they want to
+        network = build_npm_app(network, args)
+
+        # Needs to happen before any other call to set_js_runtime_options
+        # to properly test the default values, which should not emit
+        # error details on response (or in the log).
+        network = test_js_exception_output(network, args)
```

Comment:
Nit. New line to make clear which block the comment refers to?

Same at line 1463